README.md

```diff
@@ -26,7 +26,9 @@ A data package is an abstraction that encapsulates and automates data preparatio
 1. `pip install quilt`
 
 ## Linux
-1. `sudo apt-get install libhdf5-serial-dev`
+1. Install the OpenSSL headers:
+   * Ubuntu: `sudo apt-get install libssl-dev`
```

Comment:
The `cryptograpy` package, used by `requests`, only has wheels for Win and Mac: https://pypi.python.org/pypi/cryptography

Honestly, though, running `apt-get install`/`dnf install` is so easy that I wouldn't worry about it.